contracts/marketplace/Marketplace.sol

```diff
@@ -44,18 +45,18 @@ contract Marketplace {
     token.manualTransfer(seller, msg.sender, amountToBuy);
 
     //Transfer ETH to the seller
-    msg.sender.transfer(msg.value);
+    sellP.ownerAddress.transfer(msg.value);
```

Comment:
Can simply use 'seller' instead of 'sellP.ownerAddress' and remove that field from sellPosition.